ci-operator/step-registry/ipi/install/libvirt/ipi-install-libvirt-commands.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=$(yq r "${SHARED_DIR}/cluster-config.yaml" 'OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE')
+if [[ -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
+  echo "OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is an empty string, exiting"
+  exit 1
+fi
+
+echo "Installing from release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME}/${BUILD_ID}
+export HOME=/tmp
+
+# Debug echo "Cluster_type=${CLUSTER_TYPE}"
+case "${CLUSTER_TYPE}" in
+libvirt-s390x) export KUBECONFIG=${HOME}/.kube/config ;;
+libvirt-ppc64le) export KUBECONFIG=${HOME}/.kube/config ;;
+*) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+esac
```

Comment:
Why not fail early as you did in another script below? If you're setting the same value for Z and P, there is no need for the `case` here.